TweetBook/Contract/V1/ApiRoutes.cs

```diff
@@ -11,6 +11,10 @@ public static class Posts
         {
             public const string GetAll = Base +"/posts";
 
+            public const string Update = Base + "/{postId}";
```

Comment:
I see that you have 3 routes which look the same. I get that each has a different purpose, but what do you think about grouping them in the same variable and maybe naming it accordingly? Something like 'PostIdOperation'?